src/referencePackageSourceGenerator/ReferencePackageSourceTask/GenerateProject.cs

```diff
@@ -135,12 +135,19 @@ public override bool Execute()
                     }
                 }
 
+                propertiesByTfm += $"  <PropertyGroup Condition=\" '$(TargetFramework)' == '{targetFramework}' \">\n";
                 if (subPath == "lib")
                 {
-                    outputPathByTfm += $"  <PropertyGroup Condition=\" '$(TargetFramework)' == '{targetFramework}' \">\n";
-                    outputPathByTfm += $"    <OutputPath>$(ArtifactsBinDir){relativePath}/{subPath}/</OutputPath>\n";
-                    outputPathByTfm += $"  </PropertyGroup>\n\n";
+                    propertiesByTfm += $"    <OutputPath>$(ArtifactsBinDir){relativePath}/{subPath}/</OutputPath>\n";
                 }
+                if (targetFramework == "netstandard2.0" ||
```

Comment:
netstandard2.0 doesn't use implicit framework references. Why do you need to set `DisableImplicitFrameworkReferences` to false for it? I thought that you mentioned in your PR summary that you can't globally set the property to false for all TFMs exactly because of netstandard2.0.

Reply:
I did mention that, didn't I? 🫤

So it looks like `netstandard2.0` TFM for Microsoft.Bcl.AsyncInterfaces.6.0.0 will result in the following error if `DisableImplicitFrameworkReferences` is `true`:

```
error CS0234: The type or namespace name 'CompilationRelaxationsAttribute' does not exist in the namespace 'System.Runtime.CompilerServices' (are you missing an assembly reference?)
```

Setting `DisableImplicitFrameworkReferences` to false will make those errors go away.

However, for `netstandard2.0` of System.Runtime.CompilerServices.Unsafe.6.0.0, it will result in the following error if `DisableImplicitFrameworkReferences` is `false` (the opposite of the previous example):

```
error NETSDK1023: A PackageReference for 'NETStandard.Library' was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see https://aka.ms/sdkimplicitrefs
```

So in reality, there doesn't seem to be one solution that solves both without manual fixup.

Reply:
To me that indicates that we are missing some details here.

> Setting DisableImplicitFrameworkReferences to false will make those errors go away.

That's because you didn't to update this block: https://github.com/dotnet/source-build-reference-packages/blob/a848f811a52d75494029e663cd0a799be11744fe/src/referencePackageSourceGenerator/ReferencePackageSourceTask/GenerateProject.cs#L58-L61

 Remove the `&& !includesNetStandard21 && !includesNetCoreApp30` part in the condition as we now define the `DisableImplicitFrameworkReferences` property per TFM and don't need to conflate these together anymore.

Reply:
Thanks. That fixes it.